test/built-ins/TypedArray/prototype/with/BigInt/early-type-coercion-bigint.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2021 Igalia. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%typedarray%.prototype.with
+description: >
+  %TypedArray%.prototype.with invokes ToNumber before copying
+info: |
+  %TypedArray%.prototype.with ( index, value )
+
+  ...
+  7. If _O_.[[ContentType]] is ~BigInt~, set _value_ to ? ToBigInt(_value_).
+  8. Else, set _value_ to ? ToNumber(_value_).
+  ...
+features: [BigInt, TypedArray, change-array-by-copy]
+includes: [testBigIntTypedArray.js, compareArray.js]
+---*/
+
+testWithBigIntTypedArrayConstructors(function(TA) {
```

Comment:
Hmm, does this mean none of the other tests cover BigInt typed arrays?

Reply:
No, but of the new methods, only `with` has a special case for `BigInt`.